### DISCUSSION TITLE
Failed to upload workspace file

### DISCUSSION THREAD

**Discussion by tjdalsry** [open-webui/open-webui#3744] [General]
# Bug Report

## Description

**Bug Summary:**

If you upload a file with the file name in Korean in the workspace, 500 Error will appear and will not be uploaded.

Unknown PDF Filter Warning

**Steps to Reproduce:**

Workspace -> Documents -> Upload(File name in Korean)

**Expected Behavior:**

Upload Completed

**Actual Behavior:**
[Describe what actually happened.]

## Environment

- Open WebUI Version: 0.37

**Comment by tjbck** [comment]
If you could share your file we'd appreciate it!
